### DISCUSSION TITLE
Netdata Mobile App for Notifications released! 📱🚀

### DISCUSSION THREAD

**Discussion by hugovalente-pm** [netdata/netdata#16721] [Announcements]
<img height="500" alt="netdata-notifications-mobile-app" src="https://github.com/netdata/netdata/assets/82235632/a5aa7417-cecf-41b8-9426-aa790df6f764">

The New Year brings with it one of the most awaited features - The Netdata Mobile Notifications App! 
You can now receive and manage alert and reachability notifications from your subscribed spaces on the go.

The app is available on: 
* [Android](https://play.google.com/store/apps/details?id=cloud.netdata.android&pli=1) 
* [iOS](https://apps.apple.com/in/app/netdata-mobile/id6474659622)

Learn more about the feature on 
* [Documentation](https://learn.netdata.cloud/docs/alerting/notifications/centralized-cloud-notifications/netdata-mobile-app)
* [Youtube Video](https://www.youtube.com/watch?v=abCrs-rn97U&t=1358s) from the meetup

PS: This feature is only available on our Business subscriptions.

**Comment by ichbestimmtnicht** [comment]
1. (From [X](https://twitter.com/ichbestimmtnich/status/1768393437680738773)) The default cgroup notifications in docker environments really annoy me. I do not care about some random cgroup sha or whatever. Show me the damn container name. Clicking on the notification shows only useless information. pls reinvent here!
![image](https://github.com/netdata/netdata/assets/28549503/5e4ddfa3-0212-4f4d-9462-2fe0703f6d5a)
2. Don't try to make your notifications fancy or they will break any other integration. For example my smartwatch:
![img_20240315_133925](https://github.com/netdata/netdata/assets/28549503/fc096eb6-7b7b-4b43-8151-623cda54b95c)

**Reply by tkatsoulas** [reply]
Hello, these are two different issues. Could you please open two different bug reports. Let me give you some more context/guidance 

1. The cgroup naming issue is part of the Agent (for some reason your alert mechanism doesn't translate it. I won't start guessing what is going wrong, or try to resolve it in this thread. I will need a full bug report under netdata/netdata repo https://github.com/netdata/netdata/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=BUG_REPORT.yml&title=%5BBug%5D%3A+)
2. The push notification in a smart watch, I suppose it's really an actual bug for push notifications in devices with small screens. Feel free to open a bug report under netdata/netdata-cloud repo https://github.com/netdata/netdata-cloud/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=BUG_REPORT.yml&title=%5BBug%5D%3A+

Thank you in advance

**Reply by ilyam8** [reply]
> The cgroup naming issue is part of the Agent

I think it is not. Looks like the mobile app problem - https://github.com/netdata/netdata-cloud/issues/new/choose
